src/platform/site-wide/mega-menu/mega-menu-link-data.json

```diff
@@ -10,19 +10,19 @@
                         "links": [
                             {
                                 "text": "About VA Health Benefits",
-                                "href": "/health-care/about-va-health-benefits/"
+                                "href": "https://www.va.gov/health-care/about-va-health-benefits/"
```

Comment:
I think this is going to be weird on local machines, because I believe they will be re-written to dev.va.gov and not localhost. We might need to change that rewrite logic if it's going to affect links in the menus that we use more often.

Reply:
I added some logic to remove the `http://www.va.gov` from dev builds so it should point to the relative path: localhost locally or `dev.va.gov` when deployed. I verified it against the watch task.

Reply:
I really want to come up with a better way but I'd like to work for all of our needs vs creating yet another way to make urls (I think we have at least 2 right now).